staging/src/k8s.io/apimachinery/pkg/api/validation/objectmeta.go

```diff
@@ -189,15 +190,28 @@ func ValidateObjectMetaAccessor(meta metav1.Object, requiresNamespace bool, name
 	return allErrs
 }
 
+// ValidateInitializerName validates that the name is fully qualified
+func ValidateInitializerName(fldPath *field.Path, name string) field.ErrorList {
+	var allErrors field.ErrorList
+	if len(name) == 0 {
+		allErrors = append(allErrors, field.Required(fldPath, ""))
+	}
+	if errs := validationutil.IsDNS1123Subdomain(name); len(errs) > 0 {
+		allErrors = append(allErrors, field.Invalid(fldPath, name, strings.Join(errs, ",")))
+	}
+	if len(strings.Split(name, ".")) < 3 {
```

Comment:
`two dots` is hard to translate - I think most people would expect to see `three or more segments` or `parts`

Reply:
changed to "should be a domain with at least three segments separated by dots"